modules/assisted-installer-setting-the-cluster-details.adoc

```diff
@@ -0,0 +1,42 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: PROCEDURE
+[id='setting-the-cluster-details_{context}']
+
+= Setting the cluster details
+
+To create a cluster with the {ai-full} web user interface, use the following procedure.
+
+.Procedure
+
+. Log in to the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[RedHat Hybrid Cloud Console].
+
+. Open the menu on the top left and click *OpenShift*.
+
+. Click *Create cluster*.
```

Comment:
Getting into ``https://console.redhat.com/openshift/assisted-installer/clusters/~new`` already gets you into cluster creation page, making it redundant to do some of those steps (clicking on create-cluster, datacenter, etc.)
Did you mean ``https://console.redhat.com``?